packages/aws-cdk/lib/init-templates/sample-app/javascript/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     "strictPropertyInitialization":false,
     "allowJs": true,
     "checkJs": true,
-    "noEmit": true
+    "noEmit": true,
+    "typeRoots": ["./node_modules/types"]
```

Comment:
shouldn't that be `@types`?

Reply:
Jesus... yes it should be... you'd think I would have gotten it right on the first try.